### ISSUE TITLE
Auction won't allow some items to be sold?

### ISSUE THREAD

**Issue by torkur** [torilmud/issues#78]
It looks like Auction might use no sell flags or something?  It seems completely random and I question why it's in place.  Example:  Trollbark - Glossy Serpent Armor can be auctioned, but the Azurite Ring and the Mystical Crystal Skull do not.  This doesn't really make sense because it's not a "shop" and the other items are exactly what we'd want to sell on the auction for cash, worth more as quest items, etc.  Please take a look and if possible change auction to allow us to auction any item we'd like.

**Comment by dnavarro** [comment]
Yes, auction currently will not sell NOSELL flagged items.

There is actually a rule against selling quest items. Which applies to auction as well as mundane selling on global channels.

I will raise this topic with staff, and we will discuss whether we might be willing to change the NOSELL flag / the policy on selling quest items

**Comment by torkur** [comment]
I agree completely on non-wearable quest items such as a pile of dragon scales or similar.

The problem is that all 3 Trollbark items listed are wearable and all 3 are also quest items. That's the part that confuses me.  Maybe "wearable/has stats" is the enhancement I'm thinking of honestly. :)

**Comment by dnavarro** [comment]
yeah the "statless" expectation has actually been the general policy in the past. if it has actual stats then we dont consider it a "quest item" for the purposes of prohibiting selling

But that policy is entirely separate from the presence of NOSELL flags

That being said though, the azurite ring should be sellable. It doesnt have a NOSELL flag.

The crystal skull also does not have a NOSELL flag...  However its kind of a unique item in the code, so that uniqueness might be preventing auctioning. But if that's the case that should be a very rare circumstance.

Also the crystal skull /barely/ has stats, so thats kinda questionable.

**Comment by dnavarro** [comment]
Tested the ring, and I cannot auction it either. Though I don't know why...

This needs more work to find out what is happening

**Comment by Inames** [comment]
What is the identify information on a brand new azurite ring? can you post it?

**Comment by dnavarro** [comment]
```
Name 'an azurite ring'
Keyword 'ring azurite blue sparkling', Item type: TREASURE
Item can be worn on:  FINGER 
Item will give you following abilities:  NOBITS
Item is: MAGIC ANTI-EVIL LIT NO-WARRIOR NOBITS
Weight: 1, Value: 50000
Can affect you as :
    Affects : HITPOINTS By 65
    Affects : STR By 13
```

**Comment by Inames** [comment]
Could it be the item type treasure flag? normally rings are flagged item type worn

**Comment by Inames** [comment]
I don't think you can auction any "treasure" items, but I could be mistaken.

**Comment by dnavarro** [comment]
I mean, I don't see what else it could be. And if that's the case I can't imagine any purpose to prohibiting that item type...

**Comment by Inames** [comment]
I think the idea is that "Treasure" is priceless and often meant to be kept or used for something important. but no clue, i'd just conjure up a list of "treasure" items that you should otherwise be able to sell and try auctioning them to confirm. Here's a few I found on the items DB site.

a ring encrusted with tiny sapphires * -5armor 1hit * Treasure (Finger) * 0 lbs * 1000c * Zone: Silverymoon * Last ID: 8-20-08
a ring of protection from fire * 12hp fire(5%) * Treasure (Finger) * 1 lbs * 250000c * Zone: Comarian Mines (R) * Last ID: 3-1-07
a ruby studded holy symbol * 12hp 4cha * MAGIC NO-WARRIOR NO-THIEF NO-MAGE * Treasure (Hold) * 6 lbs * 80000c * Zone: Trollbark Forest (Q) * Last ID: 3-11-06
a silver rimmed monocle embedded with stardust * 14hp DET-INVIS SLOW * MAGIC NO-THIEF ANTI-CLERIC ANTI-DRUID * Treasure (Eyes) * 1 lbs * 100c * Zone: Izan's Floating Fortress (R) * Last ID: 9-6-06

**Comment by torkur** [comment]
Hey,

Just a note above. The crystal skull is the Lich item only for the quest. Trollbark item is the "mystical crystal skull" which is 26 mana 2 max power or so held psi item, so I'd expect the mystical crystal skull to have the same workings as the azurite ring. :)

Thanks!

**Comment by dnavarro** [comment]
ah, I see, yeah I was looking at the wrong skull then! Yeah that skull is also "treasure"...
